src/BlockEntities/EnchantingTableEntity.cpp

```diff
@@ -1,14 +1,14 @@
 
-// BlockEntityEnchantingTable.cpp
-
 #include "Globals.h"  // NOTE: MSVC stupidness requires this to be the same across all modules
 
 #include "EnchantingTableEntity.h"
+#include "ClientHandle.h"
```

Comment:
Uuh Tiger, didn't you get told off for doing this in another PR?

Reply:
let's change it, the ../../ are an exercise in masochism

Reply:
Open an issue or make a forum thread and get it agreed, don't just sneak it into _somebody else's_ PR!